tests/test_health.py

```
@@ -12,26 +12,26 @@
 class TestHealthValidation:
-    async def test_invalid_action(self, _mock_graphql: AsyncMock) -> None:
+    async def test_invalid_subaction(self, _mock_graphql: AsyncMock) -> None:
         tool_fn = _make_tool()
-        with pytest.raises(ToolError, match="Invalid action"):
-            await tool_fn(action="invalid")
+        with pytest.raises(ToolError, match="Invalid subaction"):
+            await tool_fn(action="health", subaction="invalid")
 
 
 class TestHealthActions:
     async def test_test_connection(self, _mock_graphql: AsyncMock) -> None:
```

Ruff (0.15.5)

[warning] Fixture `_mock_graphql` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)